Configurations.md

```diff
@@ -3096,11 +3096,11 @@ Note that no wrapping will happen if:
 1. The comment is the start of a markdown header doc comment
 2. An URL was found in the comment
 
-- **Default value**: `false`
-- **Possible values**: `true`, `false`
+- **Default value**: `"off"`
+- **Possible values**: `"doc"`, `"normal"`, `"all"` (alias `true`), `"off"` (alias `false`)
```

Comment:
Let's capitalize these to be consistent with other options and the help text defined in `ConfigType::doc_hint`.

```markdown
- **Default value**: `Off`
- **Possible values**: `Off` (alias `false`), `All` (alias `true`), `Doc`, `Normal`
```